### PR TITLE
added resource for ethical hacking

### PR DESCRIPTION
**Pull request by Shashank0271**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -442,6 +442,7 @@ Throughout this list you'll see next to each resource and emoji. Here's what eac
 - :wrench: [Open Source Reverse Engineering Platform](https://github.com/radareorg/cutter)
 - :video_camera: [Penetration Testing Bootcamp](https://hackersploit.org/penetration-testing-bootcamp/)
 - :bulb: [Practical-Ethical-Hacking-Resources](https://github.com/TCM-Course-Resources/Practical-Ethical-Hacking-Resources)
+- :video_camera: [Complete ethical hacking course](https://www.youtube.com/watch?v=tHd8k54kVs8&list=PLBf0hzazHTGOEuhPQSnq-Ej8jRyXxfYvl)
```
Comment (shahednasser) [inline README.md:445]: Replace with the URL to the playlist

**Review by shahednasser — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (Shashank0271) [reply README.md]: I have made the changes , please check .

File: README.md [hunk 2]
```diff
@@ -442,6 +442,7 @@ Throughout this list you'll see next to each resource and emoji. Here's what eac
 - :wrench: [Open Source Reverse Engineering Platform](https://github.com/radareorg/cutter)
 - :video_camera: [Penetration Testing Bootcamp](https://hackersploit.org/penetration-testing-bootcamp/)
 - :bulb: [Practical-Ethical-Hacking-Resources](https://github.com/TCM-Course-Resources/Practical-Ethical-Hacking-Resources)
+- :video_camera: [Complete ethical hacking course](https://www.youtube.com/playlist?list=PLBf0hzazHTGOEuhPQSnq-Ej8jRyXxfYvl)
```
Comment (shahednasser) [inline README.md:445]:
```suggestion
- :video_camera: [Ethical Hacking and Penetration Testing - Complete Course](https://www.youtube.com/playlist?list=PLBf0hzazHTGOEuhPQSnq-Ej8jRyXxfYvl)
```

**Review by shahednasser — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (Shashank0271) [reply README.md]: Please check , i have made the changes

Comment (Shashank0271) [reply README.md]: ?

**Pull request closed without merging**